### PR TITLE
Document 'app.focus()'

### PR DESCRIPTION
**Pull request by phamdaniel**
`app.focus()` was missing from the docs.

### CODE REVIEW

**Comment by ArekSredzki** [comment]
Closes #4719

File: docs/api/app.md [hunk 1]
```diff
@@ -265,6 +257,18 @@ Exits immediately with `exitCode`.
 All windows will be closed immediately without asking user and the `before-quit`
 and `will-quit` events will not be emitted.
 
+### `app.focus()`
+
+On Linux, focuses on the first visible window. On OSX, makes the application the active app. On Windows, focuses on the first window for the application.
```
Comment (sindresorhus) [inline docs/api/app.md]: `OSX` => `OS X`

Comment (sindresorhus) [inline docs/api/app.md]: `first window for the application` => `first window of the application`

Comment (phamdaniel) [inline docs/api/app.md]: :+1:

**Comment by zcbenz** [comment]
:+1:

**Pull request merged**